engine/wwtlib/Folder.cs

```diff
@@ -275,7 +287,10 @@ public void Refresh()
                 proxyFolder.Parent = Parent;
             }
 
-            proxyFolder.LoadFromUrl(urlField, childReadyCallback);
+            //Also listening to errors, to make sure clients do not wait forever in the case of a 404 or similar.
+            //Especially useful for recursive downloads, where potentially dussins of URL's are downloaded.
```

Comment:
Thanks, I think this will be useful to discuss in the source! English spelling: "dozens" not "dussins".

Reply:
Whoops, some Swenglish tried to sneak in...